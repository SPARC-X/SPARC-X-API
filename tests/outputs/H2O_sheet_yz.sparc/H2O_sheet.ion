#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name> 
# PSEUDO_POT: <path/to/pseudopotential/file>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...

# Reminder: when changing number of atoms, change the RELAX flags accordingly
#           as well.

ATOM_TYPE: H                 # atom type
PSEUDO_POT: ../../../psps/01_H_1_1.0_1.0_pbe_v1.0.psp8
N_TYPE_ATOM: 2               # number of atoms of this type
COORD:                       # coordinates follows
   12.556010134214192  4.265111864110533   2.834589187490385  
   12.556010134214192  1.404066510870237   2.834589187490385  

ATOM_TYPE: O               # atom type 
PSEUDO_POT: ../../../psps/08_O_6_1.2_1.4_pbe_n_v1.0.psp8
N_TYPE_ATOM: 1               # number of atoms of this type
COORD:                       # coordinates follows
   11.448630624967949  2.834589187490385   2.834589187490385   

