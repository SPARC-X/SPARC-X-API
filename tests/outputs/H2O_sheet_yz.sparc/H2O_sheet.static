***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of H:
      0.5231670889       0.7522243147       0.4999275463
      0.5231670889       0.2476307779       0.4999275463
Fractional coordinates of O:
      0.4770262760       0.4999275463       0.4999275463
Total free energy (Ha): -1.762271099378250E+01
Atomic forces (Ha/Bohr):
 -7.0091512819E-03  -2.8155137475E-02   4.7625587321E-08
 -7.0083540340E-03   2.8155728229E-02   4.1091979535E-08
  1.4017505316E-02  -5.9075452258E-07  -8.8717566856E-08
Stress (Ha/Bohr**2): 
 -5.9325459799E-03   3.7871815297E-11 
  3.7871815297E-11  -5.4929931315E-03 
Stress equiv. to all periodic (GPa): 
 -7.2725636831E+00   4.6426136346E-08 
  4.6426136346E-08  -6.7337265476E+00 
