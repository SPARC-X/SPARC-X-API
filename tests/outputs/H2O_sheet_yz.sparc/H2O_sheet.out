***************************************************************************
*                       SPARC (version Feb 03, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun Feb  5 13:17:48 2023                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 24 5.67 5.67 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 120 29 29
FD_ORDER: 12
BC: D P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.003674932
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 8
CHEB_DEGREE: 35
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-09
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: H2O_sheet/temp_run3/H2O_sheet
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
24.000000000000000 0.000000000000000 0.000000000000000 
0.000000000000000 5.670000000000000 0.000000000000000 
0.000000000000000 0.000000000000000 5.670000000000000 
Volume: 7.7157360000E+02 (Bohr^3)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 8
NP_DOMAIN_PARAL: 6 1 1
NP_DOMAIN_PHI_PARAL: 12 2 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing in x-direction        :  0.2 (Bohr)
Mesh spacing in y-direction        :  0.195517 (Bohr)
Mesh spacing in z-direction        :  0.195517 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  H2O_sheet/temp_run3/H2O_sheet.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  8
Atom type 1  (valence electrons)   :  H 1
Pseudopotential                    :  ../psps/01_H_1_1.0_1.0_pbe_v1.0.psp8
Pseudocharge radii of atom type 1  :  4.20 4.30 4.30 (x, y, z dir)
Number of atoms of type 1          :  2
Atom type 2  (valence electrons)   :  O 6
Pseudopotential                    :  ../psps/08_O_6_1.2_1.4_pbe_n_v1.0.psp8
Pseudocharge radii of atom type 2  :  7.20 7.23 7.23 (x, y, z dir)
Number of atoms of type 2          :  1
Estimated total memory usage       :  59.00 MB
Estimated memory per processor     :  1.23 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -5.8502567922E+00        4.363E-01        0.175
2            -5.8736011131E+00        8.444E-02        0.081
3            -5.8741485773E+00        4.037E-02        0.090
4            -5.8742064274E+00        2.879E-02        0.076
5            -5.8742349219E+00        9.541E-03        0.074
6            -5.8742365448E+00        2.953E-03        0.064
7            -5.8742369371E+00        1.289E-03        0.066
8            -5.8742369712E+00        3.459E-04        0.058
9            -5.8742369911E+00        1.962E-04        0.056
10           -5.8742369936E+00        7.025E-05        0.039
11           -5.8742369876E+00        9.685E-06        0.034
12           -5.8742369950E+00        2.589E-06        0.031
13           -5.8742369979E+00        7.358E-07        0.023
Total number of SCF: 13    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -5.8742369979E+00 (Ha/atom)
Total free energy                  : -1.7622710994E+01 (Ha)
Band structure energy              : -4.0591778476E+00 (Ha)
Exchange correlation energy        : -4.9328873198E+00 (Ha)
Self and correction energy         : -2.6913043403E+01 (Ha)
-Entropy*kb*T                      : -1.7023417939E-07 (Ha)
Fermi level                        : -1.8886200437E-01 (Ha)
RMS force                          :  2.4015614396E-02 (Ha/Bohr)
Maximum force                      :  2.9014859269E-02 (Ha/Bohr)
Time for force calculation         :  0.017 (sec)
Maximum stress                     :  5.9325459799E-03 (Ha/Bohr**2)
Maximum stress equiv. to periodic  :  7.2725636831E+00 (GPa)
Time for stress calculation        :  0.031 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  1.390 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
